src/LtiServiceConnector.php

```diff
@@ -87,19 +86,21 @@ public function makeServiceRequest(array $scopes, $method, $url, $body = null, $
             'Accept' => $accept,
         ];
 
-        if ($method === 'POST') {
-            $headers = array_merge($headers, ['Content-Type' => $contentType]);
-
-            $response = $this->client->request($method, $url, [
-                'headers' => $headers,
-                'json' => $body,
-                'timeout' => 60,
-            ]);
-        } else {
-            $response = $this->client->request($method, $url, [
-                'timeout' => 60,
-                'headers' => $headers,
-            ]);
+        switch ($method) {
```

Comment:
It might be worth converting the string to uppercase, so that it's not case sensitive.